.github/workflows/tuist.yml

```diff
@@ -26,111 +26,143 @@ concurrency:
-            # 'cache-xcframeworks',
-            'dependencies',
-            'edit',
-            'generate-1',
-            'generate-2',
-            'generate-3',
-            'generate-4',
-            'generate-5',
-            'generate-6',
-            'generate-7',
-            'generate-8',
-            'graph',
-            'init',
-            'list-targets',
-            'plugins',
-            'precompiled',
-            'run',
-            'scaffold',
-            'tasks',
-            'test',
-            'plugin',
+            'Build',
+            'GenerateOne',
           ]
-    needs: release_build
     steps:
-      - uses: actions/checkout@v3
-      - name: Initialize submodules
-        run: |
-          git submodule update --init fixtures/tuist_plugin
+      - uses: actions/checkout@v4
+      - uses: asdf-vm/actions/install@v2
       - name: Select Xcode
         run: sudo xcode-select -switch /Applications/Xcode_$(cat .xcode-version).app
       - uses: actions/cache@v3
-        name: 'Cache Tuist .build folder'
+        name: 'Cache vendor .build folder'
         with:
-          path: .build
-          key: ${{ runner.os }}-${{ hashFiles('.xcode-version') }}-spm-v1-${{ hashFiles('Package.resolved') }}-git-${{ github.sha }}
+          path: vendor/.build
+          key: ${{ runner.os }}-${{ hashFiles('.xcode-version') }}-spm-v1-${{ hashFiles('vendor/Package.resolved') }}
           restore-keys: |
-            ${{ runner.os }}-${{ hashFiles('.xcode-version') }}-spm-v1-${{ hashFiles('Package.resolved') }}
-            ${{ runner.os }}-${{ hashFiles('.xcode-version') }}-spm-v1
-      - uses: ruby/setup-ruby@v1
-        with:
-          ruby-version: ${{ env.RUBY_VERSION }}
+            ${{ runner.os }}-${{ hashFiles('.xcode-version') }}-spm-v1-${{ hashFiles('vendor/Package.resolved') }}
       - uses: actions/cache@v3
+        name: 'Cache fetched dependencies folder'
         with:
-          path: vendor/bundle
-          key: ${{ runner.os }}-${{ env.RUBY_VERSION }}-gems-${{ hashFiles('Gemfile.lock', 'project/cloud/Gemfile.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-${{ env.RUBY_VERSION }}-gems-
-      - name: Bundle install
-        run: |
-          bundle config path vendor/bundle
-          bundle install --jobs 4 --retry 3
-      - name: Run tests
+          path: Tuist/Dependencies/SwiftPackageManager/.build
+          key: spm-v1-${{ hashFiles('Tuist/Package.resolves') }}
+          restore-keys: spm-v1-${{ hashFiles('Tuist/Package.resolved') }}
+      - name: Fetch dependencies
+        run: tuist fetch
+      - name: Build xcbeautify
+        run: swift build --package-path vendor --product xcbeautify
+      - name: Run acceptance tests
         if: ${{ (github.event.pull_request.head.repo.full_name == github.repository) }}
-        run: FEATURE=features/${{ matrix.feature }}.feature make tuist/acceptance-test
+        run: tuist test Tuist${{ matrix.feature }}AcceptanceTests --no-cache
```

Comment:
`make tuist/test` now would run these? If yes I think we should split it in `make tuist/test/unit` and `make tuist/test/acceptance`

Reply:
`make tuist/test` already runs integration tests which takes ages, so I'm not sure if it's worth splitting. Locally, you should use `tuist test` and only pick the test suites relevant to your PR.